docs/tests-ui.rst

```diff
@@ -1,177 +1,151 @@
 
-    ./node_modules/.bin/intern-runner config=intern-browserstack
+Pass the desired URL to the command with ``--base-url``::
 
-.. _command line arguments:
+   $ py.test tests/functional/ --base-url https://developer-local.allizom.org --driver Chrome --driver-path /path/to/chromedriver
 
-Command-Line Arguments
-~~~~~~~~~~~~~~~~~~~~~~
+Run the tests in parallel
+-------------------------
 
-* ``b`` - browsers to run (e.g., ``b=chrome,firefox``)
-* ``t`` - test suites to run (e.g., ``t=wiki,homepage``)
-* ``d`` - domain to run against (e.g., ``d=developer.allizom.org``)
-* ``u`` - username for Persona (e.g., ``u=testuser@example.com``)
-* ``p`` - password for Persona (e.g., ``p=testpass``)
-* ``wd`` - slug of existing article to test (e.g., ``wd=My_Test_Doc``)
-* ``destructive=true`` - create real docs (do not run this on production)
+By default the tests will run one after the other but you can run several at
+the same time by specifying a value for `-n`::
 
-Adding a Test Suite
--------------------
+   $ py.test tests/functional/ -n auto --driver Chrome --driver-path /path/to/chromedriver
 
-To add a test suite, place your JavaScript file within the `tests/ui/tests` directory. Use the following as a template for your test suite code::
+Run tests on SauceLabs
+----------------------
 
-    define([
-        'intern!object',
-        'intern/chai!assert',
-        'base/_config'
-    ], function(registerSuite, assert, config) {
+Running the tests on SauceLabs will allow you to test browsers not on your host
+machine.
 
-        registerSuite({
+#. `Signup for an account`_.
 
-            // Unique, short name for test suite
-            name: '',
+#. Log in and obtain your Remote Access Key from user settings.
 
-            // Anything to run before each test (setup)
-            before: function() {
+#. Run a test specifying ``SauceLabs`` as your driver, and pass your credentials
+   and the browser to test::
 
-            },
+   $ SAUCELABS_USERNAME=thedude SAUCELABS_API_KEY=123456789 py.test tests/functional/ --driver SauceLabs --capability browsername MicrosoftEdge
 
-            // Text decribing what the test is testing
-            '': function() {
+Alternatively you can save your credentials `in a configuration file`_ so you
+don't have to type them each time.
 
-            }
-        });
+.. _`Signup for an account`: https://saucelabs.com/opensauce/
+.. _`in a configuration file`: http://pytest-selenium.readthedocs.io/en/latest/user_guide.html#sauce-labs
 
-    });
+Marks
+=====
 
+- nondestructive
 
-To run your new tests with, add the new suite path to the `tests/ui/_tests.js` file.
+   Tests are considered destructive unless otherwise indicated. Tests that
+   create, modify, or delete data are considered destructive and should not be
+   run in production.
 
-Identifying Test Failures
--------------------------
+- smoke
+
+   These tests should be the critical baseline functional tests.
```

Comment:
Same here - remove one space or the "definition" will be a block quote:

<img width="757" alt="screenshot 2016-11-22 12 17 02" src="https://cloud.githubusercontent.com/assets/286017/20536156/99c924be-b0ad-11e6-8dd6-db481f742048.png">